lib/sycamore/sycamore/transforms/standardizer.py

```diff
@@ -283,3 +283,19 @@ def __init__(
         **kwargs,
     ):
         super().__init__(child, f=standardizer.standardize, args=path, kwargs=kwargs)
+
+class IgnoreErrors(standardizer)
```

Comment:
This isn't quite the right syntax, so I don't think this will work.

@bsowell My guess is that we would like to have this logic be more generic, perhaps a subclass of `Map` that swallows errors from the function being applied?